example_queries/group_by/count.cc

```diff
@@ -0,0 +1,15 @@
+// udf_type: Aggregation
+// id: count
+// return_type: int
+
+class count {
+public:
+  int operator()(std::string height) {
```

Comment:
Why is height a string here, instead of an integer?

Reply:
Property is stored using string type. I chose to do so for the simplicity of coding. 
https://github.com/anirudhSK/dyn_tracing/blob/43af6932295b411b7436409e826f914612a4cee1/wasm/graph_utils.h#L17

We can convert it back to integer type or modify property map to store typed values.

Reply:
OK. Let's leave it this way for now and leave a TODO to use typed values subsequently.

Reply:
Ended up adding type conversion statement.